add-workers/du1-fec1/overlay/blazer/kustomization.yaml

```diff
@@ -38,4 +38,4 @@ patches:
     patch: |-
       - op: replace
         path: /metadata/labels/cluster-name
-        value: 'blazer-cars-lab'
+        value: 'blazer-nmstate'
```

Comment:
I am having trouble remembering why we don't specify the cluster.domain name here and instead use **cluster-nmstate**.  I always liked this to make things consistent over the numerous patches.  It's not worth nitpicking over, but an observation nonetheless.

Reply:
I tend to agree. But since this is already what is configured in the `blazer` cluster I had to change it accordingly

Reply:
got it.